README.md

```diff
@@ -23,10 +23,9 @@ With the Code4z extension pack you can:
 - Issue TSO commands.
 
 ## What’s New
-* 2020-12-18 - [Debugger for Mainframe](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.debugger-for-mainframe) (1.4) now supports multiple-user sessions and paragraph breakpoints.
-* 2020-12-17 - [Zowe Explorer](https://marketplace.visualstudio.com/items?itemName=Zowe.vscode-extension-for-zowe) (1.11.1) now supports login and logout functions for base profiles.
-* 2020-12-16 - [HLASM Language Support](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.hlasm-language-support) (0.12.0) adds diagnostic suppression for files that have no processor group configuration.
-* 2020-12-04 - [COBOL Language Support](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.cobol-language-support) (0.16.1) now supports subroutines used in the program and includes code snippets for EXEC CICS and EXEC SQL statements.
+* 2021-04-08 - [Debugger for Mainframe](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.debugger-for-mainframe) (1.5) now supports call and statement trace.
+* 2021-03-26 - [Zowe Explorer](https://marketplace.visualstudio.com/items?itemName=Zowe.vscode-extension-for-zowe) (1.13.1), see [Changelog](https://marketplace.visualstudio.com/items/Zowe.vscode-extension-for-zowe/changelog) for a list of fixes and additions.
+* 2021-03-15 - [COBOL Langage Support](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.cobol-language-support) (0.18.0) now supports copybooks outside the current workspace.

```

Comment:
there is much more and I am not sure the one you listed is the most important.. counting also 0.17.0 version we have:

- Syntax analysis for DB2 SQL statements
- Support for copybooks outside the current workspace
- Support for IDMS DML commands and statements
- Java 8 support
and maybe a link to changelog as for ZE?

Reply:
eh. the only reason I put a link to the changelog for ZE was because I had no idea which of those features listed were the most important.
the point of this was not to turn the "whats new" section into yet another changelog and knowing which features to mention in one sentence is starting to get a bit difficult. would you accept just having a link to each changelog as I've done for ZE?

Reply:
you're right...having another link in what's new section is a bit silly. It looks like "in this section, we want to tell what news we have. To see what news we have click here" :-D 
Ok, what if we replace the links of extensions in 'what's new' section with link to changelogs?
And for the ZE we can quickly check with Jelly, what items she would like to list there.